docs/topics_5/proc_web-troubleshooting-common-issues.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-troubleshooting-common-issues_{context}"]
+= Common issues when troubleshooting a {WebName} installation on OpenShift Container Platform
+
+.`NoRouteToHostException` error
+
+The following error might appear in the `mta-web-console-executor` pod log:
+
+----
+13:44:03,501 SEVERE [org.jboss.windup.web.messaging.executor.ExecutorBootstrap] (main) Could not start messaging listener due to: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]: javax.naming.CommunicationException: Failed to connect to any server. Servers tried: [http-remoting://192.0.2.4:8080 (java.net.NoRouteToHostException: No route to host)]
+----
+
+This error message is displayed in a new deployment because the `mta-web-console-executor` pod starts before the `mta-web-console` pod. Wait for the `mta-web-console` pod to start.
+
+.A pod reports insufficient resources
+
+After attempting to deploy {WebNameTitle} on OpenShift, a pod is unable to start, and the following error is seen in the `Events` tab:
+
+[source,options="nowrap"]
+----
+0/9 nodes are available: 4 Insufficient cpu, 4 MatchNodeSelector, 9 Insufficient memory.
+----
+
+The resource quota for the OpenShift project has been met and the pod is unable to obtain the requested resources.
+
+.Procedure
+
+. Do either of the following:
+
+* Increase the quota for the OpenShift project. For additional information on OpenShift quotas, see link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[Quotas and Limit Ranges] and link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[Setting Limit Ranges].
+* Reduce the requested resources for the {WebName} OpenShift project. It is recommended to have at least 2 CPUs and 4 GB of memory for the project.
```

Comment:
Remove "OpenShift" from "OpenShift" project. It is not the web console OpenShift project. It is the web console project in OpenShift.

I really dislike "Do either of the following". It just looks bad, like the doc can't make up its mind. I would change it to a single sentence.

You can either increase the resource quotas[link] and limit ranges[link] or you can reduce the requested resources of the project. The project must have at least 2 CPUs and 4 Gi memory.

Note: The memory request units are `Gi`, not `GB`. This is a common mistake. 
https://docs.openshift.com/container-platform/4.6/applications/quotas/quotas-setting-per-project.html